### PR TITLE
v0.9.1 rc-1

### PR DESCRIPTION
**Pull request by brouillette**
Search page patches
- added 'back to top' button
- added 'loading' status to results card
- patched card height jumpiness

### CODE REVIEW

File: src/js/components/Search/Search.js [hunk 1]
```diff
@@ -33,6 +39,17 @@ const Search = () => {
...
       </Row>
+      <Button
+        type="primary"
+        onClick={scrollToTop}
+        size="large"
+        shape="round"
+        loading={isFetchingData}
+        className="floating-search"
+      >
+        <UpOutlined />
```
Comment (davidlougheed) [inline src/js/components/Search/Search.js:50]: use `<Button icon={<UpOutlined />} ...>...` instead

File: src/js/components/Search/Search.js [hunk 2]
```diff
@@ -13,11 +14,16 @@ const Search = () => {
   const { t } = useTranslation();
   const dispatch = useDispatch();
   const searchSections = useSelector((state) => state.query.querySections);
+  const isFetchingData = useSelector((state) => state.query.isFetchingData);
 
   useEffect(() => {
     dispatch(makeGetKatsuPublic());
   }, []);
 
+  const scrollToTop = () => {
```
Comment (davidlougheed) [inline src/js/components/Search/Search.js:23]: move this outside of component (so below the last import line) to avoid redefining it every render

**Review by davidlougheed — CHANGES_REQUESTED** [review]
minor changes requested; other than those looks good!

**Review by gsfk — APPROVED** [review]
Works well for me. Had to make some Dockerfile changes to get it run on 2.8, but that's outside the scope of this pr.

**Review by v-rocheleau — APPROVED** [review]
Looks good.
UX comment: maybe only display "back to top" when scroll position is not already at the top

**Review by davidlougheed — APPROVED** [review]

**Pull request merged**